Code/Editor/EditorViewportWidget.cpp

```diff
@@ -2836,17 +2602,26 @@ bool EditorViewportWidget::GetActiveCameraState(AzFramework::CameraState& camera
     if (m_pPrimaryViewport == this)
     {
         cameraState = GetCameraState();
-
         return true;
     }
 
     return false;
 }
 
+void EditorViewportWidget::OnStartPlayInEditorBegin()
+{
+    m_playInEditorState = PlayInEditorState::Starting;
+}
+
 void EditorViewportWidget::OnStartPlayInEditor()
 {
+    m_playInEditorState = PlayInEditorState::Started;
+
     if (m_viewEntityId.IsValid())
     {
+        // Note that this is assuming that the Atom camera components will share the same view ptr
```

Comment:
They do, we've taken steps to ensure that in the camera component. 😄

Reply:
Indeed, I've become very familiar with the camera component and have seen that this is the case. While I was here, I wanted to added just a bit of comments for any parts of EditorViewportWidget which I thought might be confusing for future readers.